README.md

```diff
@@ -1,155 +1,152 @@
 
-**EOSCore.Shutdown()** will run all the necessary cleanup steps to shut down the SDK. It requires the generated PlatformInterface that was handed to you from Init() so that it can release the memory allocation for your session. Additionally, it dynamically unloads the libraries that were linked in Init() by using Bindings.Unhook. If you used Init() you must use Shutdown() at the end of your application lifecycle, typically in OnApplicationQuit or OnDestroy if you only have one scene.
+**EOSCore.Shutdown()** performs all the necessary cleanup steps to shut down the SDK. It requires the **PlatformInterface** generated from `Init()` to release memory allocated for the session. Additionally, it dynamically unloads the libraries linked during initialization using `Bindings.Unhook()`. If you used `Init()`, ensure you call `Shutdown()` at the end of your app’s lifecycle, typically in `OnApplicationQuit` or `OnDestroy`.
 
-### EOSAuth Layer
-
-EOSAuth provides helpful shortcuts for working with EOS accounts and the user systems. There are 2 main login systems for EOS: Auth and Connect. 
-
-**EOSAuth.LoginAuth()** lets you login a user via the Auth system. The Auth system relates to Epic accounts only. Provide the custom EOSLoginAuthSet class with the Credentials filled out and LoginAuth() will run the async operation within the SDK. To get the result, pass in the OnLoginCallback which is invoked when the operation is complete. Based on the ResultCode, Login may have been successful or failed due to various reasons. Epic recommends that you attempt multiple types of Login for convenience, i.e. Persistent Auth to see if a session is already stored -> Exchange Code coming from the launcher itself -> AccountPortal as a fallback which opens up a web browser, with each type only activating if the previous failed. Once you get a result, you can grab the given EpicAccountId as a reference to your Epic user. Learn more about the Auth Interface [here](https://dev.epicgames.com/docs/epic-account-services/auth/auth-interface).
+---
 
-For the Exchange Code Login type, you will need to grab the user token argument from the command line. Use **EOSEnv.GetExchangeCodeToken()** to quickly grab this token from the System Environment. Note that this will definitely fail in the Editor (because the token is handed to your application via the Epic Games Store launcher) so it is up to your EOS Manager to determine when to use this Login type.
+### EOSAuth Layer
 
-Note that in order to use the "Developer" login type to test your login system, you will need to run the DevAuthTool provided by the SDK [which you can download from your developer portal](https://dev.epicgames.com/docs/epic-online-services/eos-get-started/services-quick-start#step-2---download-the-eos-sdk). The program is typically located in the "tools" folder of the SDK archive. Be sure to grab it from the same version of the SDK that OpenEOS targets.
```

Comment:
I'm not sure what is happening here since it is hard to see the changes but many of EOS Auth API details seem to have been removed. Since there is no documentation, this README needs to serve as a reference for the helper functions. I'm okay with phrasing updates, but I ordered them in a specific way so that encapsulating functions can build off what the user has read previously. Would prefer to leave the core layout of this section intact for now